packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -119,11 +120,20 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
       };
     }
 
-    async createTexture(uri: string, type: string = 'image/png'):
-        Promise<ModelViewerTexture|null> {
+    async createTexture(canvas: HTMLCanvasElement):
+        Promise<ModelViewerTexture|null>;
+    async createTexture(uri: string, type?: string):
+        Promise<ModelViewerTexture|null>;
```

Comment:
I think these are unnecessary; let's just stick with broadening the accepted input types.